CODE-OF-CONDUCT.md

```diff
@@ -0,0 +1,35 @@
+
+We encourage you to confront someone or a group of people that you observe to be in violation of the CoC (whether in letter or in spirit) with your observation, so that the person(s) can correct his/her/their behavior. If this is not successful or it is difficult for you to confront the people and you seek support for this, you may contact the Code of Conduct Committee at [conduct@lists.scs.community](mailto:conduct@lists.scs.community). If you have questions, concerns, or any other inquiries please feel free to contact the project lead at [project@scs.sovereignit.de](mailto:project@scs.sovereignit.de).
+
+A large fraction of SCS consists of contributions to upstream projects. All SCS contributors are expected to adhere to the respective upstream Codes of Conduct when interacting with such projects, or developing code intended for upstreaming.
+
+## Our Standards
+
+We ask you to please adhere to the following basic rules:
+
+* **Be friendly and patient.** We were all new or suffered from a lack of knowledge at one point in time. Please try to remember what it felt like to be on that end, and treat people accordingly.
+* **Use welcoming and inclusive language.** We strive to be a community that welcomes and supports people of all backgrounds and identities. This includes, but is not limited to members of any race, ethnicity, culture, national origin, organization, color, immigration status, social and economic class, educational level, sex, sexual orientation, gender identity and expression, age, size, family status, political belief, religion, and mental and physical ability.
+* **Be helpful.** By helping others to learn our entire ecosystem is enriched. We encourage members of the SCS community to mentor each other and help to raise the general level of knowledge in the community whenever possible.
+* **Be careful in the words that you choose.** We are a community of professionals, and we conduct ourselves professionally. Be kind to others. Do not insult or put down other participants. Harassment and other exclusionary behavior aren’t acceptable. This includes, but is not limited to:
+  * Violent threats or language directed against another person.
+  * Discriminatory jokes and language.
+  * Posting sexually explicit or violent material.
+  * Posting (or threatening to post) other people’s personally identifying information (“doxing”), regardless of whether it is publicly available.
+  * Personal insults, especially those using racist, sexist, or otherwise discriminatory terms.
+  * Deliberately referring to others by names or pronouns counter to their identity.
+  * Unwelcome sexual attention.
+  * Advocating for, or encouraging, any of the above behavior.
+  * Repeated harassment of others. In general, if someone asks you to stop, then stop.
```

Comment:
Is there a specific reason why this bullet point is below the point right above? That makes the "any of the above" not cover this one.

Reply:
The harassment sentence should be moved up.
But "In general, if someone asks " does fit in the logic of listing misbehaviors. This would have to be reworded into "Ignoring that someone asks you to stop." to fit in. (I guess it was added in later during the drafting process of the original doc.) If we wanted to keep the statement "if someone asks you to stop, then stop." as general advice, we should have it below the list ...